src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -24,7 +24,7 @@ class TaskDeploymentService {
 
         val alb: LoadBalancer = LoadBalancer.builder().targetGroupArn(targetGroupArn).containerPort(containerPort).build()
 
-        val serviceBuilder = CreateServiceRequest.builder().cluster(ecs_cluster_name).loadBalancers(alb).serviceName("${user_name}_test").taskDefinition(configurationService.getStringConfig(ConfigKey.USER_CONTAINER_TASK_DEFINITION)).loadBalancers(alb).desiredCount(1).build()
+        val serviceBuilder = CreateServiceRequest.builder().cluster(ecs_cluster_name).loadBalancers(alb).serviceName("${user_name}-ui-service").taskDefinition(configurationService.getStringConfig(ConfigKey.USER_CONTAINER_TASK_DEFINITION)).loadBalancers(alb).desiredCount(1).build()
```

Comment:
I don't think `ui-service` is a good name for these services as none of it's containers serve up a UI to the user.

Reply:
I thought that JupyterHub was being served through headless chrome as the UI?

Reply:
Sure, but I wouldn't call the combination of guacd, chrome and jupyterhub a UI service. It's more like an `analytical-workspace` or something like that